src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    //constructor
```

Comment:
I noticed that you did not commnet the constructor in another classes, maybe you  canremove this comment or comment on other class constructor